### PR TITLE
CORE-7717 Update ontology AVUs to add label AVU to hierarchy AVU

### PR DESCRIPTION
**Pull request by ash3rz**
These are the UI changes to complement CORE-7715 (PR #231)

### CODE REVIEW

File: ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java [hunk 1]
```diff
@@ -69,7 +71,16 @@ public Avu convertHierarchyToAvu(OntologyHierarchy hierarchy) {
         Avu avu = avuFactory.getAvu().as();
         avu.setAttr(getAttr(hierarchy));
         avu.setValue(hierarchy.getIri());
-        avu.setUnit(hierarchy.getLabel());
+        avu.setUnit("");
```
Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java]: It looked to me like Unit wasn't optional, so I'm passing an empty string here. Let me know if this needs to change.

Comment (psarando) [inline ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java]: That should be correct: the `Unit` should be required, even if it's an empty string.

**Comment by psarando** [comment]
Looks good to me (pretty easy change too)! 👍 
#226 was just merged, which renamed `Avu.get/setAttr` to `get/setAttribute`, so you may want to rebase before merging this change, to ensure your changes are compatible.

**Comment by ash3rz** [comment]
Good call - I'll rebase, fix any issues, and then close.  Thanks for the review!

**Pull request merged**